scripts/templates/bookmark.handlebars

```diff
@@ -0,0 +1,17 @@
+<li
```

Comment:
Yes you can go this `.hbs` also. Although As we are going to use EJS this is pointless ^^!